1_beta20 fails to build in Ostro

https://ostroproject.org/jenkins/job/build_intel-corei7-64/2214/console

```
00:06:39.544 Log data follows:
00:06:39.544 | DEBUG: Executing python function sysroot_cleansstate
00:06:39.544 | DEBUG: Python function sysroot_cleansstate finished
00:06:39.544 | DEBUG: Executing shell function do_configure
00:06:39.544 | NOTE: make oldconfig
00:06:39.544 |   HOSTCC tools/kconfig/conf.o
00:06:39.544 |  GEN tools/kconfig/zconf.tab.c
00:06:39.544 |  GEN tools/kconfig/zconf.lex.c
00:06:39.544 |  GEN tools/kconfig/zconf.hash.c
00:06:39.544 |   HOSTCC tools/kconfig/zconf.tab.o
00:06:39.544 |   HOSTCC tools/kconfig/conf
00:06:39.545 | tools/kconfig/conf  --oldconfig Kconfig
00:06:39.545 | make: tools/kconfig/conf: Command not found
00:06:39.545 | tools/kconfig/Makefile:84: recipe for target 'oldconfig' failed
00:06:39.545 | make: *** [oldconfig] Error 127
00:06:39.545 | ERROR: oe_runmake failed
00:06:39.545 | ERROR: Function failed: do_configure (log file is located at /var/lib/jenkins/ostro-worker-17-slot-1-gqul3/ostro-os/build/tmp-glibc/work/corei7-64-ostro-linux/soletta/1_beta20+gitAUTOINC+dd5bc81d8d-r0/temp/log.do_configure.14166)
00:06:39.545 NOTE: recipe soletta-1_beta20+gitAUTOINC+dd5bc81d8d-r0: task do_configure: Failed
00:06:39.545 ERROR: Task 3864 (/var/lib/jenkins/ostro-worker-17-slot-1-gqul3/ostro-os/meta-soletta/recipes-soletta/soletta/soletta_git.bb, do_configure) failed with exit code '1'
00:06:59.713 NOTE: recipe linux-yocto-4.4.13+gitAUTOINC+01ac19ede0_c49ed01a7e-r0: task do_kernel_configme: Succeeded
```

--- a/recipes-soletta/soletta/soletta_git.bb
+++ b/recipes-soletta/soletta/soletta_git.bb
@@ -87,6 +87,7 @@ B = "${WORKDIR}/git"
 
 do_configure_prepend() {
    export TARGETCC="${CC}"
+   export HOSTCC="gcc"
    export TARGETAR="${AR}"
    export LIBDIR="${libdir}/"
 }

